services/backend/src/commands/System/about.ts

```diff
@@ -43,18 +51,46 @@ export class StatsCommand extends Command {
       (line) => `${Emojis.bulletPoint} ${line}`
     );
 
+    const commandStats =
+      await this.container.client.prisma.commandStats.findMany();
+
+    const formatted = commandStats
+      .sort((a, b) => b.uses - a.uses)
+      .map((c, i) => `${i + 1}. ${c.name}: ${c.uses}`)
+      .filter((_, i) => i < 5);
+
     embed.addFields([
       {
         name: "Growth (all shards)",
         value: growth.join("\n"),
       },
+      {
+        name: "Top 5 Commands:",
+        value: formatted.join("\n"),
+      },
     ]);
 
-    embed.setFooter({
```

Comment:
Why remove this? This is the only way the user can see what shard their guild has. Useful for debugging in the future.